README.md

```diff
@@ -29,9 +29,14 @@ into becoming reviewers and maintainers.
 Porter is an open-source project and things get done as quickly as we have
 motivated contributors working on features that interest them. 😉
 
+We use a single [project board][board] across all of our repositories to track
+open issues and pull requests. See Find an Issue
```

Comment:
Missing a link for `Find an Issue`?

Reply:
I was tempted to for a second but then didn't because I wanted to send new contributors through the link above to the New Contributor Guide. That sets that up for success better than linking them to issues. Plus this section is more for nosy people who want to know what we are up to.

What do you think? I am very conflicted. 😝

Reply:
I see... maybe remove `See Find an Issue` then?  It might be confusing to reference something that doesn't exist on this page and/or isn't linked to.  Either way, defer to your judgement.

Reply:
gah! thank you for making me use my brain _and_ my eyeballs. 🤦‍♀